packages/console-sdk-webpack-helpers/src/renderWebpackMicroApp.ts

```diff
@@ -46,7 +46,7 @@ export function getSDK(props: QiankunProps): {
 
 export const decorateLifecycleFunction = (lifecycleFunction: IViteParams[keyof IViteParams]) => {
   return (props: QiankunProps): void => {
-    const { isConnectedToConsole, consoleSDK } = getSDK(props)
+    const { isConnectedToConsole, consoleSDK } = getSDK(props as IConsoleProps)
```

Comment:
Why here it's required to cast as IConsoleProps?

Reply:
I remove Quiankun props from the abstract SDK. 
This caused a rewrite of types in multiple parts.
Quiankun to work needs, for lifecycle functions with QuiankunProps as params. 
This QuiankunProps contains a generic interface like this:

```
{
  container: HTML node,
  [x: string]: any
}
``` 

Inside Quiankun props we have the console props like resourceAPI, hotkeysContext & etc..
I didn't manage to create an interface with generic to include Quiankun & ConsoleProps.

I try to set up the IConsoleProps as the example below
```
const type IConsoleProps<T> = T & {...otherConsoleProps} 
```
but not work. IConsoleProps is a interface of "microfrontend-sdk" and we wouldn't use dependency like Quiankun. Have you any suggestions?

Reply:
I agree with you. But I don't understand, what qiankun props are needed from Console SDK? If all, should we use `QiankunProps & ConsoleProps`?

Reply:
we use "name" and "container" from Quiankun props.

Reply:
> I agree with you. But I don't understand, what Qiankun props are needed from Console SDK. If all, should we use `QiankunProps & ConsoleProps`?

This solution does not work. Quiankun's props are too generic.
I do the same techniques in the backend service using the decoration module "QuiankunProps"